src/components/global-header/global-header-test.stories.tsx

```diff
@@ -46,3 +46,10 @@ export const MenuWithIconOnlyButtonsStory: ComponentStory<
   );
 };
 MenuWithIconOnlyButtonsStory.storyName = "menu with icon-only buttons";
+MenuWithIconOnlyButtonsStory.parameters = {
+  docs: {
+    description: {
+      story: "Disclaimer - This example is a non-accessible use case.",
```

Comment:
Thanks @robinzigmond, I agree the documentation around the story parameters API is not the most clear, I did spend quite a bit of time trying to get this to work. I eventually found [this example](https://storybook.js.org/docs/react/writing-docs/doc-block-description#working-with-the-docspage) in Storybook's documentation showing how to add a story description via CSF. 

Given the description is not visible in our rendered docs, I assume this is an unresolved issue with storybook itself. However, I would prefer keeping a disclaimer for this story, since I have had questions from other people as to why this story is not public. I will change the disclaimer wording though as I agree your suggestion is more explicit